updating/troubleshooting_updates/restoring-cluster-previous-state.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="restoring-cluster-previous-state"]
+= Restoring your cluster to a previous state
+include::_attributes/common-attributes.adoc[]
+:context: troubleshooting_updates
+
+toc::[]
+
+If your upgrade fails, something critical has been deleted, or if your cluster experiences quorum loss there are steps to restore your cluster to a previous state. 
+
+For steps to restore your cluster to a previous state, see "Restoring to a previous cluster state".
+
+[role="_additional-resources"]
+[id="additional-resources_gathering-cluster-data"]
+.Additional resources
```

Comment:
If you choose not to go with my suggestion to remove the additional resources section and just add the xref inline, a few changes would need to be made here to better align with our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections):

1. The string that follows the underscore in the ID [doesn't uniquely describe the assembly topic and/or match the assembly context](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies). 
2. Since this is an Additional resources section for the assembly, and not for a module, the section should be formatted with == to make it a heading rather than a title.

Here's what that would look like:
```suggestion
[id="additional-resources_restoring-cluster-previous-state"]
== Additional resources
```